Test `test_parallel_replicas_insert_select/test.py::test_insert_select[test_1_shard_3_replicas_1_unavailable-3-False-3]` is flaky

### Describe the bug

https://s3.amazonaws.com/clickhouse-test-reports/json.html?REF=master&sha=122dfc8a9351ce591572b99baf06ffe78aa22d41&name_0=MasterCI&name_1=Integration%20tests%20%28aarch64%2C%20distributed%20plan%2C%201%2F4%29


### Error message and/or stacktrace

```
=================================== FAILURES ===================================
_____ test_insert_select[test_1_shard_3_replicas_1_unavailable-3-False-3] ______
[gw2] linux -- Python 3.10.12 /usr/bin/python3

start_cluster = <helpers.cluster.ClickHouseCluster object at 0xff42045dbb50>
cluster_name = 'test_1_shard_3_replicas_1_unavailable'
max_parallel_replicas = 3, local_pipeline = False, executed_queries = 3

    @pytest.mark.parametrize(
        "cluster_name,max_parallel_replicas,local_pipeline,executed_queries",
        [
            pytest.param("test_1_shard_3_replicas", 2, False, 3),
            pytest.param("test_1_shard_3_replicas", 2, True, 2),
            pytest.param("test_1_shard_3_replicas", 3, False, 4),
            pytest.param("test_1_shard_3_replicas", 3, True, 3),
            pytest.param("test_1_shard_3_replicas_1_unavailable", 3, False, 3),
            pytest.param("test_1_shard_3_replicas_1_unavailable", 3, True, 2),
            pytest.param("test_1_shard_3_replicas_1_unavailable", 2, False, 3),
            pytest.param("test_1_shard_3_replicas_1_unavailable", 2, True, 2),
        ],
    )
    def test_insert_select(start_cluster, cluster_name, max_parallel_replicas, local_pipeline, executed_queries):
        populate_count = 1000000
    
        source_table = "t_source"
        create_tables(source_table, populate_count=populate_count, skip_last_replica=False)
        target_table = "t_target"
        create_tables(target_table, populate_count=0, skip_last_replica=False)
    
        query_id = str(uuid.uuid4())
        node1.query(
            f"INSERT INTO {target_table} SELECT * FROM {source_table}",
            settings={
                "parallel_distributed_insert_select": 2,
                "enable_parallel_replicas": 2,
                "max_parallel_replicas": max_parallel_replicas,
                "cluster_for_parallel_replicas": cluster_name,
                "parallel_replicas_insert_select_local_pipeline": local_pipeline,
                "enable_analyzer": 1,
            },
            query_id=query_id
        )
>       assert (
            node1.query(
                f"select count() from {target_table}"
            )
            == f"{populate_count}\n"
        )
E       AssertionError: assert '508480\n' == '1000000\n'
E         - 1000000
E         + 508480
```

--- a/tests/integration/test_parallel_replicas_insert_select/test.py
+++ b/tests/integration/test_parallel_replicas_insert_select/test.py
@@ -92,6 +92,7 @@ def test_insert_select(start_cluster, cluster_name, max_parallel_replicas, local
         },
         query_id=query_id
     )
+    node1.query(f"SYSTEM SYNC REPLICA {target_table} LIGHTWEIGHT")
     assert (
         node1.query(
             f"select count() from {target_table}"

